user_scanner/community/stackoverflow.py

```diff
@@ -0,0 +1,35 @@
+from user_scanner.core.orchestrator import generic_validate
+from user_scanner.core.result import Result
+
+def validate_stackoverflow(user: str) -> Result:
+    url = f"https://stackoverflow.com/users/filter?search={user}"
+
+    def process(response):
+        if response.status_code == 200:
+            text = response.text
+
+            if "No users matched your search." in text:
+                return Result.available()
+
+            pattern = f'>{user}<'
+            if pattern in text:
```

Comment:
Does lowering both `pattern` and `text` improve result accuracy?

```suggestion
            if pattern.lower() in text.lower():
```